scripts/confirm-rhosls-running.sh

```
@@ -11,4 +11,4 @@ while true; do
 done
 
 # Wait for the CSV to succeed
-oc wait csv $csv --for=jsonpath='{.status.phase}'=Succeeded --timeout=300s -n openshift-lightspeed
+oc wait csv $csv --for=jsonpath='{.status.phase}'=Succeeded --timeout=300s -n openstack-lightspeed
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)